github/search.go

```diff
@@ -54,6 +55,37 @@ func (s *SearchService) Repositories(query string, opt *SearchOptions) (*Reposit
 	return result, resp, err
 }
 
+// SingleCommitResult represents a commit object as returned in commit search endpoint response.
+type SingleCommitResult struct {
```

Comment:
What do you think of following the style set forward by `CodeResult` and name this `CommitResult`?

```Go
// CodeResult represents a single search result.
type CodeResult struct {
```

Source: https://github.com/google/go-github/blob/c9c37fd6ce0247943d1aa34fc3813c62c5cb0225/github/search.go#L115-L116.

The prefix `Single` doesn't seem to be needed, is it? Unless it's needed, it's better to remove it to have a simpler struct name.